src/schemas/execute.yaml

```diff
@@ -0,0 +1,121 @@
+BatchCallArgs:
+  title: Arguments for batch call
+  type: object
+  required:
+    - block
+    - calls
+  properties:
+    block:
+      title: Block tag
+      $ref: '#/components/schemas/BlockNumberOrTag'
+    stateOverrides:
+      title: State overrides
+      $ref: '#/components/schemas/StateOverrides'
+    calls:
+      title: List of calls
+      $ref: '#/components/schemas/Calls'
+Calls:
+  title: List of message calls
+  type: array
+  items:
+    $ref: '#/components/schemas/Call'
+Call:
+  title: Call + block overrides
+  type: object
+  required:
+    - transactionArgs
+  properties:
+    transactionArgs:
+      title: Call arguments
+      $ref: '#/components/schemas/GenericTransaction'
+    blockOverrides:
+      title: Overrides for block metadata
+      $ref: '#/components/schemas/BlockOverrides'
```

Comment:
This feels like it should be part of `BatchCallArgs` not `Call`.  It feels like all of the transactions in the batch should be executed as part of a single block built on top of the target block, but with the blockOverrides embedded in the individual transactions each transaction could have different block header details. This also means that weird things could happen like time going backward between transactions which may be very difficult to implement correctly in some clients.

Reply:
Done.